EasyPost.Net/EasyPost.Net.csproj

```diff
@@ -0,0 +1,54 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <OutputType>Library</OutputType>
+    <RootNamespace>EasyPost</RootNamespace>
+    <AssemblyName>EasyPost.Net</AssemblyName>
+    <Configurations>Release;Debug;Test</Configurations>
+    <Platforms>AnyCPU</Platforms>
+    <PackageId>EasyPost.Net</PackageId>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug</OutputPath>
+    <DefineConstants>TRACE;DEBUG;NET45</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>..\lib\net</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <DocumentationFile>..\lib\net\EasyPost.Net.XML</DocumentationFile>
+    <NoWarn>1701;1702;1591;1570</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <SignAssembly>false</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Test' ">
+    <SignAssembly>false</SignAssembly>
+  </PropertyGroup>
```

Comment:
This is dead code now.

Reply:
Resolved in [50391d3](https://github.com/EasyPost/easypost-csharp/pull/209/commits/50391d3b4a6bbb7f699b8326350bc8be5afd02e9)